### ISSUE TITLE
Alllow to change prompt using a bash command

### ISSUE THREAD

**Issue by edi9999** [junegunn/fzf#3030]
I'm using fzf 0.34.0 (can't seem to update with the ~/.fzf/install --bin command...)

I would like to add some dynamism in my prompt, I usually show the time the data on that prompt was last updated.

So basically my prompt is "5 minutes ago", and after hitting a reload keybinding, I would like to change that to "30 seconds ago", or "2 minutes ago".

But obviously I don't know how "fresh" the data is, so I would like to have a way to use a bash command in the "--change-prompt" option.

**Comment by junegunn** [comment]
Invoking an external command just for the prompt feels a little bit too much to me. If you use `reload` bindings, my advice is to `--header-lines` for that functionality.

```sh
(date; echo; ps -ef) | fzf --header-lines=3 --reverse --bind 'space:reload:date | lolcat; echo; ps -ef'
```

Also, note that the header section can correctly display ANSI colors, so,

```sh
# With https://github.com/busyloop/lolcat
(date | lolcat -f; echo; ps -ef) | fzf --header-lines=3 --reverse --bind 'space:reload:date | lolcat -f; echo; ps -ef'
```

**Comment by junegunn** [comment]
While I still think using header lines is a better option, #3094 can be used in this case.

```sh
curl -XPOST localhost:6266 -d "change-prompt:$(date)> "
```

**Comment by junegunn** [comment]
Closed in 62c7f59b94ed5f3d8770a735c107233c6518acfb
